testplugins/src/main/java/org/spongepowered/test/myhomes/MyHomes.java

```diff
@@ -67,61 +68,67 @@
     public static Key<ListValue<UUID>> FRIENDS = DummyObjectProvider.createExtendedFor(Key.class, "FRIENDS");
 
     @Inject private PluginContainer container;
-    private DataRegistration<FriendsData, ImmutableFriendsData> FRIENDS_DATA_REGISTRATION;
-    private DataRegistration<HomeData, ImmutableHomeData> HOME_DATA_REGISTRATION;
+    @Inject private Logger logger;
+
+    private DataRegistration<FriendsData, ImmutableFriendsData> friendsDataRegistration;
+    private DataRegistration<HomeData, ImmutableHomeData> homeDataRegistration;
 
     @Listener
-    public void onKeyRegistration(GameRegistryEvent.Register<Key<?>> evnet) {
+    public void onKeyRegistration(GameRegistryEvent.Register<Key<?>> event) {
+        this.logger.info("onKeyRegistration");
         DEFAULT_HOME = Key.builder()
             .type(new TypeToken<Value<Home>>() {})
             .id("myhomes:default_home")
             .name("Default Home")
             .query(DataQuery.of("DefaultHome"))
             .build();
+        event.register(DEFAULT_HOME);
```

Comment:
Unless you changed the `SpongeKeyBuilder` implementation, this is redundant and should throw an exception for attempting double registration.